spec/models/submission_file_spec.rb

```diff
@@ -15,4 +15,16 @@
       expect(SubmissionFile.new.filename).to be_nil
     end
   end
+
+  describe '#temporary_download_url' do
+    it 'returns the temporary download URL of the attachment' do
+      submission_file = FactoryBot.create(:submission_file, :with_attachment, filename: 'not-really-an.xls')
+
+      expect(submission_file.temporary_download_url).to match(/not-really-an.xls$/)
```

Comment:
could we make this matcher a bit more specific to signify it's an actual URL with the right host at least?